manifests/t/TheBrowserCompany/Arc/0.15.2.18950/TheBrowserCompany.Arc.installer.yaml

```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: TheBrowserCompany.Arc
+PackageVersion: 0.15.2.18950
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.22000.0
+InstallerType: msix
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.4

```

Comment:
```suggestion
  - PackageIdentifier: Microsoft.WindowsAppRuntime.1.5
```

```
2024-03-29 13:42:08.881 [CORE] Deployment operation #1: Windows cannot install package TheBrowserCompany.Arc_0.15.2.18950_x64__ttt1ap7aakyb4 because this package depends on a framework that could not be found. Provide the framework "Microsoft.WindowsAppRuntime.1.5" published by "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", with neutral or x64 processor architecture and minimum version 5001.70.1338.0, along with this package to install.
2024-03-29 13:42:08.881 [FAIL] C:\__w\1\s\external\pkg\src\AppInstallerCommonCore\Deployment.cpp(161)\WindowsPackageManager.dll!00007FFBE6C176FF: (caller: 00007FFBE6B597DD) Exception(1) tid(1fe0) 80073CF3 Package failed updates, dependency or conflict validation.
```